tapchannel/commitment.go

```diff
@@ -652,8 +662,12 @@ func CreateAllocations(chanState *channeldb.OpenChannel, ourBalance,
 			"allocations: %w", err)
 	}
 
+	log.Tracef("Channel initiator's asset balance is %d",
+		initiatorAssetBalance)
+
 	// Next, we add the HTLC outputs, using this helper function to
 	// distinguish between incoming and outgoing HTLCs.
+	var haveHtlcSplitRoot bool
```

Comment:
I think we should add a comment here to explain how `haveHtlcSplitRoot` is used and why it exists.